mockgcp/common/operations/operations.go

```diff
@@ -125,7 +125,12 @@ func (s *Operations) StartLRO(ctx context.Context, metadata proto.Message, callb
 }
 
 func rewriteTypes(any *anypb.Any) {
-	// Fix our mockgcp hack
+	// Fix our mockgcp hack:
+	// The protobuf libraries get upset if we have two proto message types
+	// with the same proto path, but different go paths.
```

Comment:
> Is it because we have a copy of the same proto message type in the protobuf lib to implement the mock and caused the potential conflict?

Right.

> How is the proto path computed?

It's defined in the proto file.

> Do we own the two proto message types? Or just one of them?

One is in the built-in proto library.  One is the one we used to generate grpc-gateway.  Now I'm wondering if we can use the existing types with grpc-gateway.  But I'll look at that in a separate PR if that's OK, this isn't something we're introducing in this PR.